pkg/sources/git/git.go

```diff
@@ -58,14 +58,15 @@ type Git struct {
 	verify             bool
 	metrics            metrics
 	concurrency        *semaphore.Weighted
+	skipBinaries       bool
 }
 
 type metrics struct {
 	commitsScanned uint64
 }
 
 func NewGit(sourceType sourcespb.SourceType, jobID sources.JobID, sourceID sources.SourceID, sourceName string, verify bool, concurrency int,
-	sourceMetadataFunc func(file, email, commit, timestamp, repository string, line int64) *source_metadatapb.MetaData,
+	sourceMetadataFunc func(file, email, commit, timestamp, repository string, line int64) *source_metadatapb.MetaData, skipBinaries bool,
```

Comment:
This function just keeps growing 🤣 

Not something to fix in this PR of course, but I'm looking forward to refactoring this.

Reply:
Yea... it was a bit painful adding another param :(